From a75b0a6a7ef6e14e3b65b34bbc6935491d3b016e Mon Sep 17 00:00:00 2001
From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>
Date: Fri, 14 Dec 2018 17:02:07 +0000
Subject: [PATCH 2/8] compat: Generic HW_COMPAT_RHEL7_6

RH-Author: Dr. David Alan Gilbert <dgilbert@redhat.com>
Message-id: <20181214170211.14267-3-dgilbert@redhat.com>
Patchwork-id: 83516
O-Subject: [RHEL8 qemu-kvm PATCH v3 2/6] compat: Generic HW_COMPAT_RHEL7_6
Bugzilla: 1655820
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>

From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>

Add the HW_COMPAT_RHEL7_6 macro based on the 2.12 and 3.0 macros.

Signed-off-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 include/hw/compat.h | 29 +++++++++++++++++++++++++++++
 1 file changed, 29 insertions(+)

diff --git a/include/hw/compat.h b/include/hw/compat.h
index f08cc7c..40db243 100644
--- a/include/hw/compat.h
+++ b/include/hw/compat.h
@@ -513,5 +513,34 @@
         .value    = "off",\
     },
 
+/* The same as HW_COMPAT_3_0 + HW_COMPAT_2_12
+ * except that
+ *   there's nothing in 3_0
+ *   migration.decompress-error-check=off was in 7.5 from bz 1584139
+ *
+ */
+#define HW_COMPAT_RHEL7_6 \
+    { /* HW_COMPAT_RHEL7_6 from HW_COMPAT_2_12 */ \
+        .driver   = "hda-audio",\
+        .property = "use-timer",\
+        .value    = "false",\
+    },{ /* HW_COMPAT_RHEL7_6 from HW_COMPAT_2_12 */ \
+        .driver   = "cirrus-vga",\
+        .property = "global-vmstate",\
+        .value    = "true",\
+    },{ /* HW_COMPAT_RHEL7_6 from HW_COMPAT_2_12 */ \
+        .driver   = "VGA",\
+        .property = "global-vmstate",\
+        .value    = "true",\
+    },{ /* HW_COMPAT_RHEL7_6 from HW_COMPAT_2_12 */ \
+        .driver   = "vmware-svga",\
+        .property = "global-vmstate",\
+        .value    = "true",\
+    },{ /* HW_COMPAT_RHEL7_6 from HW_COMPAT_2_12 */ \
+        .driver   = "qxl-vga",\
+        .property = "global-vmstate",\
+        .value    = "true",\
+    },
+
 
 #endif /* HW_COMPAT_H */
-- 
1.8.3.1

